packages/driver/cypress/integration/e2e/multi-domain/multi_domain_yield_spec.ts

```diff
@@ -0,0 +1,234 @@
+      }
+    }).should('equal', '')
+  })
+
+  it('succeeds if subject cannot be serialized and is not accessed', () => {
+    cy.switchToDomain('foobar.com', () => {
+      cy
+      .get('[data-cy="dom-check"]')
+    }).then((obj) => {
+      return 'object not accessed'
+    }).should('equal', 'object not accessed')
+  })
+
+  it('throws if subject cannot be serialized and is accessed', (done) => {
+    cy.on('fail', (err) => {
+      assertLogLength(logs, 7)
+      expect(logs[6].get('error')).to.eq(err)
+      expect(err.message).to.include('`cy.switchToDomain()` could not serialize the subject due to one of it\'s properties not being supported by the structured clone algorithm.')
+
+      done()
+    })
+
+    cy.switchToDomain('foobar.com', () => {
+      cy
+      .get('[data-cy="dom-check"]')
+    }).should('equal')
+  })
+
+  it('throws if an object contains a function', (done) => {
+    cy.on('fail', (err) => {
+      assertLogLength(logs, 7)
+      expect(logs[6].get('error')).to.eq(err)
+      expect(err.message).to.include('`cy.switchToDomain()` could not serialize the subject due to one of it\'s properties not being supported by the structured clone algorithm.')
+
+      done()
+    })
+
+    cy.switchToDomain('foobar.com', () => {
+      cy.wrap({
+        key: () => {
+          return 'whoops'
+        },
+      })
+    }).invoke('key').should('equal', 'whoops')
+  })
+
+  it('throws if an object contains a symbol', (done) => {
+    cy.on('fail', (err) => {
+      assertLogLength(logs, 7)
+      expect(logs[6].get('error')).to.eq(err)
+      expect(err.message).to.include('`cy.switchToDomain()` could not serialize the subject due to one of it\'s properties not being supported by the structured clone algorithm.')
+
+      done()
+    })
+
+    cy.switchToDomain('foobar.com', () => {
+      cy.wrap({
+        key: Symbol('whoops'),
+      })
+    }).should('equal', undefined)
+  })
+
+  it('throws if an object is a function', (done) => {
+    cy.on('fail', (err) => {
+      assertLogLength(logs, 7)
+      expect(logs[6].get('error')).to.eq(err)
+      expect(err.message).to.include('`cy.switchToDomain()` could not serialize the subject due functions not being supported by the structured clone algorithm.')
+
+      done()
+    })
+
+    cy.switchToDomain('foobar.com', () => {
+      cy.wrap(() => {
+        return 'text'
+      })
+    }).then((obj) => {
+      // @ts-ignore
+      obj()
+    })
+  })
+
+  it('throws if an object is a symbol', (done) => {
+    cy.on('fail', (err) => {
+      assertLogLength(logs, 7)
+      expect(logs[6].get('error')).to.eq(err)
+      expect(err.message).to.include('`cy.switchToDomain()` could not serialize the subject due symbols not being supported by the structured clone algorithm.')
+
+      done()
+    })
+
+    cy.switchToDomain('foobar.com', () => {
+      cy.wrap(Symbol('symbol'))
+    }).should('equal', 'symbol')
+  })
+
+  // NOTE: This test will only work on chrome.
```

Comment:
What happens on the other browsers? Is there a way to only have this run on chrome so it doesn't need to be skipped?

Reply:
On other browsers the structured clone algorithm fails and the error won't be serialized.

Reply:
I've updated this test to succeed on both chromium and firefox browsers